### PR TITLE
Add status cli command

### PR DESCRIPTION
**Pull request by junchaowu**

### CODE REVIEW

File: cmd/dosa/schema.go [hunk 1]
```diff
@@ -131,6 +131,53 @@ func (c *SchemaUpsert) Execute(args []string) error {
...
+
+	ctx, cancel := context.WithTimeout(context.Background(), options.Timeout.Duration())
+	defer cancel()
+
+	status, err := client.CheckSchemaStatus(ctx, c.NamePrefix, c.Version)
+	if err != nil {
+		if c.Verbose {
+			fmt.Printf("detail:%+v\n", err)
+		}
+		fmt.Println("Status: NOT OK")
```
Comment (zhenkai) [inline cmd/dosa/schema.go]: Can we just print the error msg instead of "NOT OK"?

File: doc.go [hunk 2]
```diff
@@ -22,7 +22,9 @@
 //
 // Abstract
 //
-// [dosa][dosa-spec] is a library that provides a distributed
+// DOSA open source is not ready for prime time yet. We will announce when it's ready.
```
Comment (zhenkai) [inline doc.go]: why add this?

Comment (junchaowu) [inline doc.go]: We have a build tool to make go doc based on readme. Readme has been updated to those statement.

File: cmd/dosa/schema.go (hunk 1 above)
Comment (junchaowu) [inline cmd/dosa/schema.go]: The `err` will return to the main(). main() will return error status code.

**Pull request merged**